packages/backend/src/api/request.ts

```diff
@@ -170,7 +170,7 @@ export function buildRequest(options: CreateBackendApiOptions) {
 
       return {
         data: null,
-        errors: parseErrors(err),
+        errors: err?.errors,
```

Comment:
@Nikpolik, I am not sure I get this fix. These errors are reported on the server and don't propagate to the client where ClerkJS lives. Can you please elaborate?

Reply:
Initially parse errors is called here. This transforms it to camel case.

Then the result is passed to [`withLegacyReturn`](https://github.com/clerk/javascript/blob/nikpolik%2Fv4-fix-backend-error-message-details/packages/backend/src/api/request.ts#L61-L65) which initialises a new `ClerkAPIResponseError`
Finally parse errors is called again inside [`ClerkAPIResponseError`](https://github.com/clerk/javascript/blob/release%2Fv4/packages/shared/src/error.ts#L109-L109)